### ISSUE TITLE
Rename package

### ISSUE THREAD

**Issue by victorgarcia98** [flexiblepower/s2-python#15]
Perhaps, the name `s2-ws-json-python`package is too large and could induce some confusion as we are not using WebSockets (yet, maybe we want to add some WS capabilities) nor JSON in this package. 

A point to maintain the name is to keep a stronger link to https://github.com/flexiblepower/s2-ws-json.

Name suggestions:
* `s2-protocol`
* `s2-models`
* `s2-flexibility`

In case of having interest in addressing this, I would do it before improving the packaging and publishing it to PyPI. Otherwise, feel free to close this issue.

Please, let me know your thoughts  :smile:

**Comment by TNO-SlaFleur** [comment]
I completely agree that shorter name is needed. For context, let me share the original thoughts on why I chose the current name:

s2-ws-json-python is broken up in 4 parts:
- s2 - It is the S2 protocol, so this part makes sense.
- ws - It is the websocket version of the S2 protocol. The S2 standard allows for multiple technical implementations; it only defines the datastructures in an abstract way.
- json - Similarly, the current datastructure which is used is json, but any format is allowed.
- python - It is the python-specific implementation. While redundant when using in a python script through `import s2-ws-json-python` (no need to mention it is Python while we are writing a Python script..), I did want to allow for implementation for other languages. So when searching for the Python implementation using a search engine, I want users to find this easily; hence the addition of Python.

The same reasoning is why the current schemas are in the repo called 's2-ws-json'. It is the S2 standard using websocket transport and json datastructures. At some point (this is just an example), we may get a s2-amqp-protobuf for instance to denote an AMQP transport protocol with protobuf formatting. Personally I think it is a bad idea to allow multiple transport protocols as it will hinder adoption significantly as they are not-compatible, but the standard allows for it.

All-in-all, the current name is long, tough but very practical and still I hate it :P . I do not think we will have another implementation using a different transport protocol soon and, as you said, we have no websocket-specificness in this repo. I would like to keep 'python' in here.

Perhaps `s2-python` for short? Or `s2-protocol-python` if you feel we should include the word 'protocol?

Normally I love to keep the repo name, package name and import name the same for simplicity. But, perhaps in this case, it makes sense to split it up if see a good reason.

Curious to your thoughts!
